lib/android/app/src/main/java/com/reactnativenavigation/react/NavigationModule.java

```diff
@@ -72,6 +73,27 @@ public String getName() {
         return NAME;
     }
 
+    @ReactMethod
+    public void getLaunchArgs(String commandId,Promise promise) {
+        NavigationActivity activity = activity();
```

Comment:
NavigationModule should delegate to other entities. Please move all this logic to `LaunchArgsParser`,
Then `getLaunchArgs` will include a single line of code - `promise.resolve(LaunchArgsParser.parse(activity()))`

Reply:
done